test/auth/authentication-client.tests.js

```diff
@@ -67,8 +68,73 @@ describe('AuthenticationClient', function() {
     }
   });
 
+  describe('client info', function() {
+    it('should generate default telemetry value', function() {
+      var client = new AuthenticationClient({ token: 'token', domain: 'auth0.com' });
+      var pkgVersion = require('../../package.json').version;
+      var nodeVersion = process.version.replace('v', '');
+      expect(client.getClientInfo()).to.deep.equal({
+        name: 'node-auth0',
+        version: pkgVersion,
+        env: [{ node: nodeVersion }]
+      });
+    });
+
+    it('should configure instances with default telemetry header', function() {
+      sinon.stub(AuthenticationClient.prototype, 'getClientInfo', function() {
+        return { name: 'test-sdk', version: 'ver-123' };
```

Comment:
Why was `env` left out of this test object?

Reply:
This is just mocking a hardcoded response we can compare against in the tests. In order to avoid depending on versions numbers that can change, etc. 

But the default implementation is already being tested here https://github.com/auth0/node-auth0/pull/362/files/ebe7db56674569acdfbcb25458fbe2357e9728ca#diff-16d97c0f0c5a6d9cc470e15de5e52580R72